src/libraries/Pool.sol

```diff
@@ -328,6 +332,10 @@ library Pool {
 
         bool exactInput = params.amountSpecified < 0;
 
+        if (!exactInput && (slot0Start.swapFee == SwapFeeLibrary.MAX_SWAP_FEE)) {
+            revert CannotSpecifyOutputAmountWithMaxSwapFee();
```

Comment:
this error is a bit long, and when writing the name for the revert test I thought it could maybe be a bit more concise:

CannotSpecifyOutputAmountWithMaxSwapFee -> InvalidFeeForExactOut()

and then the test name would be more clear:

test_swap_revertsWith_InvalidFeeForExactOut_whenFeeIsMax or something like that